aztk_sdk/version.py

```diff
@@ -22,4 +22,4 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-__version__ = '0.3.1'
+__version__ = '0.4.1'
```

Comment:
0.4.0?

Reply:
We have a stable tag of 0.4.1 now. https://github.com/Azure/aztk/tree/v0.4.1